packages/library-base/proguard-rules-consumer-common.pro

```diff
@@ -102,6 +102,10 @@
     *;
 }
 
+-keep class kotlin.Unit {
```

Comment:
Why is this necessary?

Reply:
This is the return type for `Function0`. See https://github.com/realm/realm-kotlin/issues/1150#issuecomment-1348535477